corehq/apps/cloudcare/templates/formplayer/pagination.html

```diff
@@ -4,7 +4,7 @@
 
 
 {% block pagination_templates %}
-  <div class="container">
+  <div class="pagination-container">
```

Comment:
Can you add `pagination-container` as a second class rather than replacing `pagination-container`? `container` comes from bootstrap and looks like it has both its own styles and a bunch of downstream styles, we probably don't want to lose those.